website_theme_flexible/static/description/index.html

```diff
@@ -0,0 +1,216 @@
+<section id="website_theme_flexible_description" class="container">
```

Comment:
This file is not needed having README.rst

Reply:
Where is that stated?

Reply:
First: 
See point 4 from an email App Developers received beginning of the year.

From: apps@odoo.com
Sent: Thursday, 18 January, 2018 14:26:59
Subject: Let's improve the Apps Store - Apps Review
```
Dear App Publishers,

We hope this email finds you well. Back in December we shared with you our vision for the future of the Odoo Apps Store, including building more customer trust (claims, reimbursement process), improving services to vendors and differentiating the apps with more transparency and quality checks.

We also announced the first wave of App reviews based on the new automatic scoring system, which will arrange the apps in order of ranking. As a reminder, every app page of the store has been reviewed and scored based on 5 criteria, and depending on the score received, the app page will be displayed higher or lower in the default App listing.

To summarize: only 5/5 apps, fully complying with the Guidelines, can appear at the top of the default search.

Below you'll find the 5 criteria: 
1. The module has no icon 
2. The module has no cover image (thumbnail) License is not set in the module manifest 
3. The module has a rating that is quite poor (lower than 3) 
4. The module has a non-html description (like rst for example), as we'd like to promote beautiful descriptions
5. The tie breaker between apps with the same score remains the best sellers.

Please take a moment to read the above criteria and review your App pages accordingly. Upon connecting to your App Store Account and visiting the description page of each of your apps, you should see your score, if you didn't reach 5 out of 5,  and the elements that require amendments for your ranking to improve.

Thank you for all your work.

Kind regards,

Cécile Collart
Content Marketer at Odoo
```
Second: Check out how the RST is rendered in the App Store now. Already depreciated.

Reply:
Thanks for the info. This is a proposal, but not yet done. I worry about that, as an HTML is difficult to maintain. RST can be a rich format also. The way I see to go with this is to auto-generate an HTML from RST instead of manually create one, or better yet, forget totally the Odoo Apps and go for OCA Apps Browser (https://github.com/OCA/apps-store/issues/7).

Meanwhile, I won't oppose to this.

Reply:
Thanks for pointing me to the "OCA AppStore" module. That is a great development. 

However I do feel OCA developed apps should also be highly visible in the Odoo App Store. Preferably top ranked to. And aside great code, Odoo S.A. would love to see a good presentation. 

As my programming skills are sub OCA standards helping out getting the Apps presented as good as possible can be my OCA contribution ;-)

Reply:
About index.html and scoring, it is official, read Odoo vendor guidelines (look for "Scoring of Apps")
https://www.odoo.com/apps/vendor-guidelines